fix(hmr-plugin): improved hmr update state (#765)
ngxs/store: docs/plugins/hmr.md

```

Now if you make changes to one of your components, those changes should be visible automatically without a complete browser refresh.

### HMR lifecycle

> If you want to do some additional manipulations with the states during the hmr lifecycle, you can use the built-in actions. However, this does not work for production.

Reviewer: ```suggestion
If you want to do some modifications to the state during the hmr lifecycle you can use these built-in actions. They will not be executed in production.
```
